$NetBSD$

On NetBSD, existing test on bmake may not be necessary 
or even harmfull :-)

--- fbsd2pkg.orig	2015-10-07 23:58:48.000000000 +0900
+++ fbsd2pkg	2015-10-08 00:04:02.000000000 +0900
@@ -40,10 +40,6 @@ if [ $# != 2 ]; then
     usage
 fi
 
-if ! which bmake; then
-    printf "Error: bmake not found in PATH.\n"
-    exit 1
-fi
 
 port_path=$1
 maintainer=$2
@@ -56,6 +52,10 @@ fi
 if [ `uname` = NetBSD ]; then
     make=make
 else
+    if ! which bmake; then
+	printf "Error: bmake not found in PATH.\n"
+    exit 1
+    fi
     make=bmake
 fi
 
